### PR TITLE
Set document title as app name

### LINKED ISSUE #1842: [Feature] Appsmith page tab should have the app name displayed
## Summary
We should add the app name to the html title attribute in view mode

## Motivation
If you have multiple Appsmith apps open in different browser tabs, it is hard to find the one you need to click on. If the app name exists in the title, it will be quick to find

## Additional Context
Appsmith tabs in view mode have no indication about the app or page
Screenshot 2020-11-23 at 10 51 42 AM

Github, Notion are very specific about what action is there in every tab 
Screenshot 2020-11-23 at 10 53 28 AM
Screenshot 2020-11-23 at 10 51 16 AM

### PR DESCRIPTION
**Pull request by ashwanisindhu1**
Fixes #1842 

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Manually tested

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/pages/AppViewer/viewer/AppViewerHeader.tsx [hunk 1]
```diff
@@ -163,6 +164,8 @@ export const AppViewerHeader = (props: AppViewerHeaderProps) => {
   const canEdit = isPermitted(userPermissions, permissionRequired);
   const queryParams = new URLSearchParams(useLocation().search);
   const hideHeader = !!queryParams.get("embed");
+  useDocumentTitle(currentApplicationDetails?.name);
```
Comment (hetunandu) [inline app/client/src/pages/AppViewer/viewer/AppViewerHeader.tsx:167]: Use [Helmet](https://github.com/appsmithorg/appsmith/blob/release/app/client/src/pages/Editor/index.tsx#L221-L224) library here instead. 
Also If no name is found, it should have a default of `Appsmith`

**Review by hetunandu — CHANGES_REQUESTED** [review]
Use an existing library instead of setting this directly

File: app/client/src/pages/AppViewer/viewer/AppViewerHeader.tsx (hunk 1 above)
Comment (ashwanisindhu1) [reply app/client/src/pages/AppViewer/viewer/AppViewerHeader.tsx]: Updated to use Helmet. 
By default, [global title](https://github.com/appsmithorg/appsmith/blob/release/app/client/public/index.html#L9) is `Appsmith`. So, we do not need to set default here.

**Review by hetunandu — APPROVED** [review]

**Comment by mohanarpit** [comment]
@ashwanisindhu1 This PR has been merged. It will be available on Appsmith's next release. Thank you so much for your contribution to the project. 🎉

**Pull request merged**